src/components/common/animations/SpinnerCheckMark/SpinnerCheckMark.js

```diff
@@ -9,6 +9,9 @@ class SpinnerCheckMark extends React.Component {
       if (e.currentTime > 130.5 && !success) {
         this.anim.goToAndPlay(0, true)
       }
+
+      //speed up when finished
+      this.anim.setSpeed(1.5)
```

Comment:
this line set speed on every frame.
If we need only on finish we need condition
```
if (e.currentTime > 130.5 && success) {
        this.anim.setSpeed(1.5)
      }	      
```

and we need to add logic to set it only once

Reply:
fixed. doing it on componentdidupdate